### DISCUSSION TITLE
Automation the Interaction with the fiserv DNA - Quick Search Panel  (Screen Pop)

### DISCUSSION THREAD

**Discussion by ScreenPopRob** [Fiserv/Support#97] [Q&A]
Fiserv support,  
       I am writing what I will call a screen pop application that will be interacting with the fiserv DNA product.  I work for a Telecommunications/Call Center Software vendor and a customer of ours would like us to feed an account number into your application instead of  one of the call center agents having to key in the account number.   I've attached a screen shot of their front end.   From a conversation I've had with them, they click on a "Quick Search" tab and that opens up a Quick Search Panel and then they type in their account number into the Account Number Text box and then hit Enter or Click Query.

I would prefer not to interact with the UI but rather some type of Gateway/API where I could hand you the Account Number and you could present this tab.  I am getting more specific product information from them but could tell me if you have a way of interacting with your product without having to interacting with the UI.

The version of software our customer is using is:  "Our core is called Fiserv DNA. The current database version number is DNA 4.7.2.1 . "

![fiservDNAFrontEnd](https://user-images.githubusercontent.com/99829088/154329153-224e37ed-ea36-4deb-916d-465e6811c230.PNG)

**Comment by TaniaS-fiserv** [comment]
Hi @ScreenPopRob Thank you for reaching out to us! 

@bobburghardt @tsegurajimen could you please assist @ScreenPopRob with the queries!

Thanks!

**Reply by bobburghardt** [reply]
@ScreenPopRob - the APIs we have available do not support a screen pop into the cores.  Thanks for contacting us.
